pysonde/sounding.py

```diff
@@ -181,6 +181,20 @@ def get_sonde_type(self):
             logging.warning("Sonde type for METEOMODEM is assumed to be 163")
             self.meta_data["sonde_type"] = "163"
 
+    def get_sonde_serial_number(self):
+        """Get sonde serial number"""
+
+        if self.level0_reader == "MW41":
+            # Check if "SerialNbr" exists in meta_data
+            if "SerialNbr" in self.meta_data:
+                self.meta_data["sonde_serial_number"] = self.meta_data["SerialNbr"]
+            else:
+                logging.warning("Serial number missing in meta_data for MW41.")
+                self.meta_data["sonde_serial_number"] = np.nan  # Assign NaN if missing
```

Comment:
The comment states the obvious so we can remove it to keep it cleaner
```suggestion
                self.meta_data["sonde_serial_number"] = np.nan
```